bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/config/player-config.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:freeboxos:player">
+		<parameter name="macAddress" type="text" pattern="([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})">
+			<label>MAC Address</label>
+			<description>The MAC address of the player device</description>
+		</parameter>
+		<parameter name="id" type="integer" required="true">
+			<label>ID</label>
+			<description>Id of the player</description>
+			<default>1</default>
```

Comment:
Does it make sense to define a default id ?
In this case, please remove the tag "required".
But it makes probably more sense to keep the tag "required" and remove the default value.

Reply:
The freebox player, defaults to 1 (unless you've got many and dished your first one I suppose), so it kind of makes sense.

Reply:
The case you have two players is not very rare. But ok, if the id is always 1 when you have only one player, you can keep the default value.